### ISSUE TITLE
About Dynamo package install

### ISSUE THREAD

**Issue by kibeom22** [DynamoDS/DynamoRevit#2788]
If this issue is **not** a bug report or improvement request, please check the [Dynamo forum](https://forum.dynamobim.com/), and start a thread there to discuss your issue.

## Dynamo version
Dynamo Core : 2.10.1.3976
Dynamo Revit : 2.10.1.4002
(Which version of Dynamo are you using? Go to Help > About if you're not sure.)

## Revit version
Revit 2022
(Which version of Revit are you using?)

## Operating system
Window 10
(e.g. Windows 7, Windows 8.1, etc)

## What did you do? 
To use Dynamo, I tried to download the most installed package on the Dynamo site. However, some packages contain "Python scripts or binarys in one of the packages or items that are dependent on the package. Do you want to continue?" There was an error. example, "archi-lab.net", "Rhythm", "bimorphNodes" etc. If you look at the details of the error, "Adll file found at C:\ProgramData\Autodec\RVT 2022\Dynamo\2.10\packages\archi-lab.It says net\bin\archilab2022.dll did not have a signed certificate."
(Fill in here)

## What did you expect to see?
I want to install a normal package.
(Fill in here)

## What did you see instead?

(Fill in here)

**Comment by QilongTang** [comment]
@saintentropy @mjkkirschner do you know how to address this?

**Comment by mjkkirschner** [comment]
yes, the %programData% folder is somehow first in the users package paths list, move it down.

**Comment by kibeom22** [comment]
> yes, the %programData% folder is somehow first in the users package paths list, move it down.

What does "move it down" mean? I deleted the (((C:\ProgramData\Autodec\RVT 2022\Dynamo\2.10\packages\archi-lab))) folder and downloaded it again, but if I turn off and on the Revit, there's an error again.

**Comment by QilongTang** [comment]
> > yes, the %programData% folder is somehow first in the users package paths list, move it down.
> 
> What does "move it down" mean? I deleted the (((C:\ProgramData\Autodec\RVT 2022\Dynamo\2.10\packages\archi-lab))) folder and downloaded it again, but if I turn off and on the Revit, there's an error again.

I think @mjkkirschner means to move it down in the Manage Node and Package Paths window. You can find it under Settings
![image](https://user-images.githubusercontent.com/3942418/149964301-c1050829-08e5-4de0-a890-bb834cfdcb55.png)

**Comment by kibeom22** [comment]
> > > yes, the %programData% folder is somehow first in the users package paths list, move it down.
> > 
> > 
> > What does "move it down" mean? I deleted the (((C:\ProgramData\Autodec\RVT 2022\Dynamo\2.10\packages\archi-lab))) folder and downloaded it again, but if I turn off and on the Revit, there's an error again.
> 
> I think @mjkkirschner means to move it down in the Manage Node and Package Paths window. You can find it under Settings ![image](https://user-images.githubusercontent.com/3942418/149964301-c1050829-08e5-4de0-a890-bb834cfdcb55.png)

Thank, I think the other packages have been resolved to some extent.
But as shown in the picture, the archi-lab package appears to have been installed, but if you look at add-ons, you can't see it, and the nodes of archi-lab don't appear, so why is this?[](url
![설치된 패키지 2022-01-18 오후 5_30_56](https://user-images.githubusercontent.com/97723754/150081342-cad03f15-c5ca-4764-b456-fb7fe05cffd8.png)
![Dynamo 2022-01-18 오후 5_31_02](https://user-images.githubusercontent.com/97723754/150081390-c6f80191-d0e3-4ba1-a9b8-464489160586.png)
)

**Comment by QilongTang** [comment]
@kibeom22 Would you check console on the bottom of your session for errors? I tried to install the same version of the package to Dynamo 2.13 and it gives me some errors. You may want to contact the author and report the error

**Comment by johnpierson** [comment]
Closing this out - the question was answered in the thread above and the issue was simply never closed.

If the answer no longer holds on a current build, please open a fresh issue.
